Resources/config/validator.xml

```diff
@@ -11,6 +11,10 @@
 
             <tag name="validator.constraint_validator" alias="sonata.admin.validator.inline" />
         </service>
+
+        <service id="sonata_admin.validator.validator_wrapper" class="Sonata\AdminBundle\Validator\ValidatorWrapper">
```

Comment:
Not sure if it will work, but can you try making your service private with `public="false"`? The less public services we have, the less we will have to worry about BC